go.mod

```diff
@@ -1,123 +1,140 @@
 module github.com/konveyor/openshift-velero-plugin
 
-go 1.19
+go 1.21
+
+toolchain go1.21.1
 
 require (
-	github.com/bombsimon/logrusr v1.0.0
+	github.com/bombsimon/logrusr/v3 v3.0.0
 	github.com/containers/image/v5 v5.19.0
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.3.0
 	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
 	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/openshift/library-go v0.0.0-20200521120150-e4959e210d3a
-	github.com/sirupsen/logrus v1.8.1
-	github.com/stretchr/testify v1.7.0
+	github.com/sirupsen/logrus v1.9.3
+	github.com/stretchr/testify v1.8.4
 	github.com/vmware-tanzu/velero v1.6.2
-	k8s.io/api v0.20.7
-	k8s.io/apimachinery v0.20.7
-	k8s.io/client-go v0.20.7
+	k8s.io/api v0.29.3
```

Comment:
I feel like this might cause an issue with type conversions if any .

I would add replace for all k8s.io that originated from the same kubernetes/kubernetes monorepo

Reply:
Like velero 1.7 using one thing, but openshift-velero-plugin return a different thing

Reply:
That's a 9 Y bump

Reply:
https://github.com/kubernetes-csi/external-snapshotter/blob/master/go.mod#L80

Been used in other places